provider/pkg/gen/typegen.go

```diff
@@ -19,9 +19,9 @@ import (
 	"encoding/json"
 	"fmt"
 	"regexp"
+	"sort"
 	"strings"
 
-	"github.com/ahmetb/go-linq"
```

Comment:
Removing `go-linq` as a dependency and using idiomatic golang for data processing instead.